PhysicsTools/SelectorUtils/interface/PFJetIDSelectionFunctor.h

```diff
@@ -416,10 +419,10 @@ class PFJetIDSelectionFunctor : public Selector<pat::Jet>  {
       quality_ = TIGHT;
       }
 
-   if( (version_ != WINTER17 && version_ != WINTER17PUPPI) && quality_ ==  TIGHTLEPVETO){
-      edm::LogWarning("BadJetIDVersion") << "JetID version does not support the TIGHTLEPVETO operating point -- defaulting to TIGHT";
-      quality_ = TIGHT;
-      }
+   //if( (version_ != WINTER17 && version_ != WINTER17PUPPI) && quality_ ==  TIGHTLEPVETO){
+   //   edm::LogWarning("BadJetIDVersion") << "JetID version does not support the TIGHTLEPVETO operating point -- defaulting to TIGHT";
+   //   quality_ = TIGHT;
+   //   }
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
This is irrelevant now, the comments were just used for testing -- taken out